schemas/core/components/terms.json

```diff
@@ -96,7 +96,7 @@
             "multipleOf": 1
           },
           "type": {
-            "enum": ["maxRate", "missedReturnPenalty"]
+            "enum": ["maxRate", "missedReturnPenalty", "perExceedingInterval"]
```

Comment:
As we discussed on slack. Let's not add new type, but instead add new optional `endAt` property which should be used instead  of `timeInterval` when we deal with fixed rate.